python/pyspark/sql/tests/test_functions.py

```diff
@@ -286,6 +286,45 @@ def test_dayofweek(self):
         row = df.select(dayofweek(df.date)).first()
         self.assertEqual(row[0], 2)
 
+    def test_date_add_function(self):
+        from pyspark.sql.functions import date_add, col
```

Comment:
you can import this on the top, and let's add a comment like # SPARK-XXXXX: blah blah (see also https://spark.apache.org/contributing.html)

Reply:
Thank you for the review @HyukjinKwon - have amended as per your comments.